blank-typescript/analytics.js

```diff
@@ -0,0 +1,38 @@
+const os = require('os')
+
+function analytics_enabled() {
+  return process.env.BOTONIC_DISABLE_ANALYTICS !== '1'
+}
+
+function getSystemInformation() {
+  return {
+    platform: os.platform(),
+    version: os.version(),
+    arch: os.arch(),
+    timezone: Intl.DateTimeFormat().resolvedOptions().timeZone,
+    timestamp: new Date().toISOString(),
+  }
+}
+
+function track(event, properties = {}) {
+  if (
+    analytics_enabled() &&
+    analytics &&
+    credentials &&
+    credentials.analytics
```

Comment:
Where `analytics` and `credentials` come from?

Reply:
thanks @asastre , i missed it! now fixed 👌